src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -248,6 +248,9 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
                     </Link>
                   </li>
                 )}
+                <li>
```

Comment:
Does this link need to be added to the Admin content page and prerelease page?

Reply:
yes that makes sense, have added it now.